### PR TITLE
Lint ignore files

### PR DESCRIPTION
**Pull request by rbuysse**

### CODE REVIEW

**Review by peterschwarz — APPROVED** [review]

File: justfile [hunk 1]
```diff
@@ -81,6 +81,7 @@ clean:
 lint:
     #!/usr/bin/env sh
     set -e
+    ./bin/lint_ignore
```
Comment (vaporos) [inline justfile:84]: This should be inlined so its easier to copy justfile between machines/branches.

File: justfile [hunk 2]
```diff
@@ -81,6 +81,7 @@ clean:
 lint:
     #!/usr/bin/env sh
     set -e
+    just lint-ignore
```
Comment (vaporos) [inline justfile:84]: This could be listed as a dependency "lint: lint-ignore"

File: justfile [hunk 3]
```diff
@@ -100,6 +101,12 @@ lint-client:
     cd examples/gameroom/gameroom-app
     npm run lint
 
+lint-ignore:
+   #!/usr/bin/env sh
+    set -e
+    diff -u .dockerignore .gitignore
+    echo "\n\033[92mLint Ignore Files Success\033[0m\n"
```
Comment (vaporos) [inline justfile:108]: This looks like it needs an if $? conditional?

Comment (vaporos) [reply justfile:108]: Maybe not

File: justfile [hunk 4]
```diff
@@ -100,6 +101,12 @@ lint-client:
     cd examples/gameroom/gameroom-app
     npm run lint
 
+lint-ignore:
+   #!/usr/bin/env sh
```
Comment (vaporos) [inline justfile:105]: Indent of this line is slightly off

**Review by peterschwarz — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Review by peterschwarz — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Pull request merged**